### ISSUE TITLE
ParseVar doesn't handle quoted strings properly

### ISSUE THREAD

**Issue by deslaughter** [OpenFAST/openfast#2552]
**Bug description**
<A clear and concise description of the bug.>

When `ParseVar` is used to read quoted strings, such as a file path, it will split the path if it contains a space even though it is within the quotes. The previous guidance has been that paths cannot contain spaces, but this seems like a simple problem to fix. 

**To Reproduce**
<Update the following list with your specific information.>
1. Specify a path containing a space inside a quoted string in the Aerodyn input file for `OLAFInputFIleName`.
2. OpenFAST will report that it can't find the file because it splits the path at the space.

**Expected behavior**
<A clear and concise description of what you expected to happen.>
OpenFAST should use the full path between the double quotes.

**OpenFAST Version**
<Please provide as much detail as possible including git commit. The best information is a screenshot of the OpenFAST system description that prints when running OpenFAST:>

```
**************************************************************************************************
 OpenFAST

 Copyright (C)  National Renewable Energy Laboratory
 Copyright (C)  Envision Energy USA LTD

 This program is licensed under Apache License Version 2.0 and comes with ABSOLUTELY NO WARRANTY.
 See the "LICENSE" file distributed with this software for details.
 **************************************************************************************************

 OpenFAST-v3.5.3
```

**Comment by andrew-platt** [comment]
We might need to add a new routine specifically for file name handling.

**Comment by andrew-platt** [comment]
Fixed with #2558
